### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by Aimeess** [sadeepj/crfasrnn_keras#66]
Hi,
I am a college student.I have used several days to run this demo.In the last steep,I use `python3 run_demo.py` ,the result are as follow:
`Using TensorFlow backend.
WARNING: Logging before flag parsing goes to stderr.
W0709 23:16:12.372608 140072129275712 deprecation_wrapper.py:119] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:74: The name tf.get_default_graph is deprecated. Please use tf.compat.v1.get_default_graph instead.

W0709 23:16:12.372946 140072129275712 deprecation_wrapper.py:119] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:517: The name tf.placeholder is deprecated. Please use tf.compat.v1.placeholder instead.

W0709 23:16:12.376471 140072129275712 deprecation_wrapper.py:119] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:4138: The name tf.random_uniform is deprecated. Please use tf.random.uniform instead.

W0709 23:16:12.395432 140072129275712 deprecation_wrapper.py:119] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3976: The name tf.nn.max_pool is deprecated. Please use tf.nn.max_pool2d instead.

W0709 23:16:12.508162 140072129275712 deprecation_wrapper.py:119] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:133: The name tf.placeholder_with_default is deprecated. Please use tf.compat.v1.placeholder_with_default instead.

W0709 23:16:12.512938 140072129275712 deprecation.py:506] From /home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
Segmentation fault (core dumped)
`
Could anyone help me?
Thanks.

**Comment by Nsquared2** [comment]
I'm having the exact same issue.

**Comment by sadeepj** [comment]
@Aimeess @Nsquared2 The exception posted by Aimeess is thrown by `/home/aimer/.local/lib/python3.6/site-packages/keras/backend/tensorflow_backend.py` (by Keras). Could you try updating your Keras version please? If it still doesn't work, please post your Keras version and I'll have a look. Code to find out the Keras version:
```
import keras
keras.__version__
```

**Comment by Aimeess** [comment]
@sadeepj My keras version is 2.2.4.At first,I use tensorflow cpu version(1.14.0),and then there is the result:Segmentation fault (core dumped).After downgrading tensorflow to version 1.10.0,the problem was solved.It also work on CUDA9.0,cudnn7.0 and tensorflow-gpu 1.9.0.
As a beginner,after getting this demo,I do not know how to train this neural network.Could you give me some instruction such as your training code and the struct of the training set.
I would appreciate it if you can help me.
My mail address is 897870186@qq.com
Thanks a lot
